src/comparisons/elements/create_elements/es6.js

```diff
@@ -0,0 +1,7 @@
+function generateElements(html) {
+  const template = document.createElement('template');
+  template.innerHTML = html.trim();
+  return template.content.firstChild;
```

Comment:
It looks like jQuery is return an HTMLCollection here, so this would be more appropriate as:
```suggestion
  return template.content.children;
```

Reply:
The HTMLCollection is more correct. That IE8 example is actually wrong here:
https://codepen.io/abbondanzo/pen/GRdRoRY

Reply:
Ok, propagated the change to that file as well.